libs/ledger/include/ledger/miner/transaction_layout_queue.hpp

```diff
@@ -44,15 +44,16 @@ class TransactionLayoutQueue
 
   /// @name Iteration
   /// @{
-  ConstIterator    cbegin() const;
-  ConstIterator    begin() const;
-  Iterator         begin();
-  ConstIterator    cend() const;
-  ConstIterator    end() const;
-  Iterator         end();
-  std::size_t      size() const;
-  bool             empty() const;
-  DigestSet const &digests() const;
+  ConstIterator                         cbegin() const;
+  ConstIterator                         begin() const;
+  Iterator                              begin();
+  ConstIterator                         cend() const;
+  ConstIterator                         end() const;
+  Iterator                              end();
+  std::size_t                           size() const;
+  bool                                  empty() const;
+  DigestSet const &                     digests() const;
+  std::unordered_set<TransactionLayout> TxLayouts() const;
```

Comment:
Worth a typedef, to be in-line with those above?

Reply:
Done.